### ISSUE TITLE
creating a fedex account

### ISSUE THREAD

**Issue by rhhayward** [alertedsnake/packagetracker#3]
Is there a process for an individual to create a fedex account for use with this project?

I know the fedex code isn't working, but I'm quite interested in using this project, and was going to tinker with it. Following the instructions, I see that a fedex key and password is required.  Therefore, I tried creating an account, organization, and project at https://developer.fedex.com/api/en-us/get-started.html, and have a testing API key/secret.  I couldn't get those to work with this package (see note below).  I went to create a production API key, but following that process, I got to the point of needing to create a corporate account.

If you can lay out the steps, I'd be happy to make a PR adding the documentation.

output when trying to use the package with testing creds:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/<user>/proj/packagetracker/packagetracker/__init__.py", line 176, in track
    return self.iface.track(self.tracking_number)
  File "/home/<user>/proj/packagetracker/packagetracker/service/fedex_interface.py", line 96, in track
    raise TrackFailed(e)
packagetracker.exceptions.TrackFailed: Authentication Failed (Error code: 1000)
```

**Comment by alertedsnake** [comment]
Hmm honestly, the last time I did it was probably 4 years ago, I have no idea what the current process is.  Though I think I did create a corporate account, not that I use it for shipping more than like once a year.

Huh though I just used this yesterday to track a package, I'll have to update the readme - I don't recall what was broken but I think it was upstream.  Regarding your error, this package is really just a wrapper around [python-fedex](https://github.com/python-fedex-devs/python-fedex), they may have better docs or more details about that error message.

Sorry I couldn't be more helpful!
